tachyon/c/crypto/random/rng.h

```diff
@@ -17,21 +31,83 @@ struct tachyon_rng {
 extern "C" {
 #endif
 
+/**
+ * @brief Creates a random number generator from the given seed.
+ * @param type The random number generator type.
```

Comment:
```suggestion
 * @param type The type of random number generator.
```
I think I've seen more you describing as "The type of XXX" below.